### DISCUSSION TITLE
block['bbox'] returning negative x0 value

### DISCUSSION THREAD

**Discussion by sunov** [pymupdf/PyMuPDF#1545] [Looking for help]
x0 =-2.4000000953674316
x1 = 802.1585693359375
y0= 527.2000122070312
y1 = 811.1312255859375

**Comment by JorjMcKie** [comment]
That's normal

**Reply by JorjMcKie** [reply]
Don't forget that a page may have different rectangles - contained in each other like MediaBox, CropBox, etc.
CropBox may be a true subset of MediaBox. Object coordinates are relative to the CropBox, This is how this phenomenon may occur.
